proguard.txt

```diff
@@ -1,10 +1,11 @@
--dontobfuscate
+-keeppackagenames com.bugsnag.**
+-keepnames class com.bugsnag.**
+-keepclassmembernames class com.bugsnag.** { *; }
 
--renamesourcefileattribute SourceFile
 -keepattributes Exceptions,InnerClasses,Signature,Deprecated,
                 SourceFile,LineNumberTable,*Annotation*,EnclosingMethod
 
--keep public class * {
+-keep public class com.bugsnag.** {
```

Comment:
Was there some history behind keeping `*` instead of just `com.bugsnag.*` here, @snmaynard?